### DISCUSSION TITLE
How to use MultiChoice to group to group existing actions?

### DISCUSSION THREAD

**Discussion by claremacrae** [chhoumann/quickadd#249] [Q&A] [answered]
I've just discovered MultiChoice, and am tying to figure out how to use it:
https://github.com/chhoumann/quickadd/blob/master/docs/Choices/MultiChoice.md

Here are my settings:

![image](https://user-images.githubusercontent.com/4840096/170866603-21d88e2c-0041-4b84-a8bf-ef748a7c6d23.png)

How can I move the New Task ones in to "Task Manager" group please?

I've tried all sorts of combinations of dragging the drag handles around, and not managed to actually group them together like in the image in the MultiChoice page.

OS: Mac
Obsidian: 0.14.15
QuickAdd 0.5.5

**Comment by AnnaKornfeldSimpson** [comment, marked as answer]
@claremacrae The following worked for me on Windows, thanks to a hint from #110.
Make sure the multi is unfolded (as it is in your screenshot). Click the drag handle of one of the choices you want to add and drag it to just below and to the right of the drag handle for the multi. When successful, the choice will be indented under the multi.
Hopefully this works for you. I wish the docs were a little clearer about this!

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Anna's drag technique was right for its era (and it is literally in the docs now), but there is an easier path today: right-click the choice and pick **Move to: Task Manager** - no dragging at all. That menu shipped in 2.10.0. Dragging itself was also reworked since and behaves much more predictably.

Docs: https://quickadd.obsidian.guide/docs/Choices/MultiChoice

Closing as resolved.
